aliyun/src/main/java/org/apache/iceberg/aliyun/oss/BaseOSSFile.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.aliyun.oss;
+
+import com.aliyun.oss.OSS;
+
+abstract class BaseOSSFile {
+  private final OSS client;
+  private final OSSURI uri;
+
+  BaseOSSFile(OSS client, OSSURI uri) {
+    this.client = client;
+    this.uri = uri;
+  }
+
+  public String location() {
+    return uri.location();
+  }
+
+  public OSS client() {
+    return client;
+  }
+
+  public OSSURI uri() {
+    return uri;
+  }
+
+  public boolean doesFileExists() {
+    return client.doesObjectExist(uri.bucket(), uri.key());
+  }
+
+  @Override
+  public String toString() {
+    return uri.toString();
```

Comment:
I think this should show that it is a file and not the location. Otherwise it is hard to distinguish what object you have in a REPL or error output. Can you add a more traditional `toString` that uses a `MoreObjects` builder?

Reply:
After rework, the output format as below:
OSSInputFile{file=oss://oss-testing-bucket-4d1ac83f-8d3f-4c67-bd84-c8e58b2e8b96/mock-objects/readfile.dat}